tests/mock_network/mod.rs

```diff
@@ -65,6 +66,55 @@ fn nodes_with_prefix_mut<'a>(
         .filter(move |node| prefix.matches(&node.name()))
 }
 
+pub fn count_sections_members_if_split(nodes: &[TestNode]) -> BTreeMap<Prefix<XorName>, usize> {
+    let mut counts = BTreeMap::new();
+    for pfx in nodes.iter().map(node_prefix_if_split) {
+        *counts.entry(pfx).or_default() += 1;
+        let _ = counts.entry(pfx.sibling()).or_default();
+    }
+    counts
+}
+
+pub fn node_prefix_if_split(node: &TestNode) -> Prefix<XorName> {
```

Comment:
this function seems better to be a helper function in node ?

Reply:
Could be but it is really specialized, not clear there would be much use in other tests. so it does not seem reasonable to pollute that interface.